tests/flytekit/unit/core/test_type_engine.py

```diff
@@ -2863,23 +2863,6 @@ def test_get_underlying_type(t, expected):
     assert get_underlying_type(t) == expected
 
 
-@pytest.mark.parametrize(
-    "t,expected",
-    [
-        (None, (None, None)),
-        (typing.Dict, ()),
-        (typing.Dict[str, str], (str, str)),
-        (
-                Annotated[typing.Dict[str, str], kwtypes(allow_pickle=True)],
-                (typing.Dict[str, str], kwtypes(allow_pickle=True)),
-        ),
-        (typing.Dict[Annotated[str, "a-tag"], int], (Annotated[str, "a-tag"], int)),
-    ],
-)
-def test_dict_get(t, expected):
-    assert DictTransformer.extract_types_or_metadata(t) == expected
-
-
```

Comment:
nit: can you remove the [other one](https://github.com/flyteorg/flytekit/blob/master/tests/flytekit/unit/core/test_generice_idl_type_engine.py#L2861-L2875) instead?

Reply:
sure